src/main/java/com/salesforce/storm/spout/dynamic/persistence/zookeeper/CuratorHelper.java

```diff
@@ -97,11 +96,9 @@ public void writeJson(final String path, final Object data) {
      * will deserialize the JSON to a concrete class.
      *
      * @param path node containing JSON to read from.
-     * @param <K> key to the json field.
-     * @param <V> value of the json field.
      * @return map representing the JSON stored within the zookeeper node.
      */
-    public <K, V> Map<K, V> readJson(final String path) {
+    public HashMap readJson(final String path) {
```

Comment:
Why change this?  Seems like its moving backwards?

Reply:
To avoid unchecked issues. The way this one works, it's going to always be a `HashMap` so the explicitness doesn't take anything away, just makes everything more specific.